test/webapps/PerfTestApp/build.gradle

```diff
@@ -1,7 +1,9 @@
 apply plugin: 'war'
 
+def aiSdkVersion = version
+
 dependencies {
-    compile aiCoreJar
+    compile "com.microsoft.azure:applicationinsights-core:$aiSdkVersion"
```

Comment:
Can we use `$version` here directly? I think it's understood when working in multi-module project that all versions are the same.

Reply:
I wanted to have the option to use a different version than the one in the repo.